### PR TITLE
update release instructions

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: docs/source/coredev/index.rst [hunk 1]
```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
...
 All pull requests should usually be made against ``master``, if a Pull Request
 need to be backported to an earlier release; then it should be tagged with the
-correct ``milestone``. We then use `ghpro <https://pypi.python.org/pypi/ghpro>`
+correct ``milestone``.
+
+If you are an admin on the IPython repository just mention the backport bot to
+do the work for you. The bot is evolving so instruction may be different. At
+the time of this writing you can use::
+
+    @meeseeksdev[bot] backport to <branchname>
```
Comment (takluyver) [inline docs/source/coredev/index.rst]: Are we meant to put `[bot]` in? I've been invoking it without that, and it seemed to work.

Comment (Carreau) [inline docs/source/coredev/index.rst]: yes it works without the `[bot]` ot the `@`. GitHub just happend to autocomplete with both when the bot is part of a conversation.

File: docs/source/coredev/index.rst [hunk 2]
```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
 
 All pull requests should usually be made against ``master``, if a Pull Request
 need to be backported to an earlier release; then it should be tagged with the
-correct ``milestone``. We then use `ghpro <https://pypi.python.org/pypi/ghpro>`
+correct ``milestone``.
+
+If you are an admin on the IPython repository just mention the backport bot to
+do the work for you. The bot is evolving so instruction may be different. At
```
Comment (takluyver) [inline docs/source/coredev/index.rst]: instructions (plural) may change.

File: docs/source/coredev/release_process.rst [hunk 3]
```diff
@@ -243,8 +246,12 @@ The IPython website should document the new release:
 Make sure to update readthedocs and set the latest tag as stable, as well as
 checking that previous release is still building under its own tag. 
 
+13. Update the Conda-Forge feedstock
+------------------------------------
+
+Follow the instruction on `the repository <https://github.com/conda-forge/ipython-feedstock>`_
```
Comment (takluyver) [inline docs/source/coredev/release_process.rst]: instructions (plural)

File: docs/source/coredev/index.rst (hunk 1 above)
Comment (takluyver) [inline docs/source/coredev/index.rst]: Thanks!

File: docs/source/coredev/index.rst [hunk 4]
```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
 
 All pull requests should usually be made against ``master``, if a Pull Request
 need to be backported to an earlier release; then it should be tagged with the
-correct ``milestone``. We then use `ghpro <https://pypi.python.org/pypi/ghpro>`
+correct ``milestone``.
+
+If you are an admin on the IPython repository just mention the backport bot to
```
Comment (willingc) [inline docs/source/coredev/index.rst]: I would bold the `backport bot` the first time it's used here so it stands out a bit.

File: docs/source/coredev/index.rst [hunk 5]
```diff
@@ -22,7 +22,21 @@ Backporting Pull requests
...
+do the work for you. The bot is evolving so instruction may be different. At
+the time of this writing you can use::
+
+    @meeseeksdev[bot] backport to <branchname>
+
+The bot will attempt to backport the current pull-request and issue a PR if
+possible.
+
+Backport with GhPro
```
Comment (willingc) [inline docs/source/coredev/index.rst]: I would use all caps or all lower case for ghpro in the title.

**Review by willingc — APPROVED** [review]
A few small suggestions. Looks good!

**Pull request merged**